introduction_to_repository.md

```diff
@@ -9,7 +9,12 @@ This document aims to provide a introduction to the repository and it's structur
 
 ### Monitoring toolset
 
-The `monitoring` directory contains a set of folders containing different test suites to test different capabilities of the DSS during development and production use.
+The `monitoring` directory contains a set of folders containing different test suites to test different capabilities of the DSS during development and production use. 
+
+### Running tools locally
+
+- When running tools in the monitoring toolset for local debugging outside of Docker, the [monitoring](https://github.com/interuss/dss/tree/master/monitoring) top-level folder must be added to your `PYTHONPATH`.
+- To run a DSS instance locally, you will need to run the `run_locally.sh` in the [/build/dev/](https://github.com/interuss/dss/blob/master/build/dev/run_locally.sh) directory. This will setup a local instance of the DSS and the Auth server at `http://localhost:8085`. 
```

Comment:
Let's refer the reader to where there is more information:

```markdown
To run a DSS instance locally, see [this documentation](build/dev/stand_alone_instance.md).
```

Reply:
Fixed this now, thanks!